packages/proto/src/lib/filter_v2.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+// 12/WAKU2-FILTER rfc: https://rfc.vac.dev/spec/12/
+
+import "message.proto";
+
+
+// Protocol identifier: /vac/waku/filter-subscribe/2.0.0-beta1
+message FilterSubscribeRequest {
+  enum FilterSubscribeType {
+    SUBSCRIBER_PING = 0;
+    SUBSCRIBE = 1;
+    UNSUBSCRIBE = 2;
+    UNSUBSCRIBE_ALL = 3;
+  }
+
+  string request_id = 1;
+  FilterSubscribeType filter_subscribe_type = 2;
+
+  // Filter criteria
+  string pubsub_topic = 10;
```

Comment:
```suggestion
  optional string pubsub_topic = 10;
```

the RFC says `optional`. But I question it: https://github.com/vacp2p/rfc/pull/562/files#r1170791687

Reply:
Note that this is only `optional` within the context of protobuf field design: some types of `FilterSubscribeRequests` (e.g. `SUBSCRIBER_PING`, `UNSUBSCRIBE_ALL`) SHOULD NOT have a populated `pubsub_topic`. Within the context of the _protocol_ itself this field is conditionally mandatory (which would require `optional` in the protobuf), e.g. the `pubsub_topic` MUST be populated, _if_ the `filter_subscribe_type` is `SUBSCRIBE` or `UNSUBSCRIBE`.

Reply:
Ah indeed thank you for that.